index.js

```diff
@@ -0,0 +1,43 @@
+var express = require('express');
+var app = express();
+var metrics = require('node-monitor-ui');
+
+const PORT = process.env.PORT || 3000;
+var USE_GCI = process.env.USE_GCI || 'FALSE';
+var MSG_SIZE = process.env.MSG_SIZE || 12;
```

Comment:
Please change the or to 1024 (like [here](https://github.com/gcinterceptor/msgpush-go/blob/master/main.go#L17))